speakers.js

```diff
@@ -0,0 +1,97 @@
+document.getElementById('spk-title1').textContent = speakers[1].name;
+document.getElementById('spk-subTitle1').textContent = speakers[1].description;
+document.getElementById('spk-text1').textContent = speakers[1].text;
+
+document.getElementById('spk-img2').src = speakers[2].picture;
+document.getElementById('spk-title2').textContent = speakers[2].name;
+document.getElementById('spk-subTitle2').textContent = speakers[2].description;
+document.getElementById('spk-text2').textContent = speakers[2].text;
+
+document.getElementById('spk-img3').src = speakers[3].picture;
+document.getElementById('spk-title3').textContent = speakers[3].name;
+document.getElementById('spk-subTitle3').textContent = speakers[3].description;
+document.getElementById('spk-text3').textContent = speakers[3].text;
+
+document.getElementById('spk-img4').src = speakers[4].picture;
+document.getElementById('spk-title4').textContent = speakers[4].name;
+document.getElementById('spk-subTitle4').textContent = speakers[4].description;
+document.getElementById('spk-text4').textContent = speakers[4].text;
+
+document.getElementById('spk-img5').src = speakers[5].picture;
+document.getElementById('spk-title5').textContent = speakers[5].name;
+document.getElementById('spk-subTitle5').textContent = speakers[5].description;
+document.getElementById('spk-text5').textContent = speakers[5].text;
+
+/*
+document.getElementById('seeTonic').addEventListener('click', () => {
+  document.getElementById('modal-container').style.display = 'block';
+
+  document.getElementById('project-name').textContent = projects[0].name;
+  document.getElementById('modal_li1').textContent = projects[0].clases1;
+  document.getElementById('modal_li2').textContent = projects[0].clases2;
+  document.getElementById('modal_li3').textContent = projects[0].clases3;
+  document.getElementById('project-image').src = projects[0].ftdImage;
+  document.getElementById('modal_li4').textContent = projects[0].technologies0;
+  document.getElementById('modal_li5').textContent = projects[0].technologies1;
+  document.getElementById('modal_li6').textContent = projects[0].technologies2;
+  document.getElementById('modal_li7').textContent = projects[0].technologies3;
+  document.getElementById('modal_li8').textContent = projects[0].technologies4;
+  document.getElementById('modal_li9').textContent = projects[0].technologies5;
+  document.getElementById('modal-description').textContent = projects[0].description;
+  document.getElementById('live-link').textContent = projects[0].liveVersion;
+  document.getElementById('source-link').textContent = projects[0].source;
+  document.getElementById('modal-close').src = projects[0].closeButton;
+  document.getElementById('liveIMG').src = projects[0].liveButton;
+  document.getElementById('srcIMG').src = projects[0].sourceButton;
+});
+
+document.getElementById('modal-close').addEventListener('click', () => {
+  document.getElementById('modal-container').style.display = 'none';
+});
+
```

Comment:
- [ ] Please remove all code you are not using from your documents. This will mak your code neat and also professional.